smos/package.yaml

```diff
@@ -50,11 +50,12 @@ library:
   - text
   - time
   - tz
-  - unix
   - unliftio
   - validity
   - vty
-
+  - vty-crossplatform
+  - vty-windows
```

Comment:
I don't think we can have `vty-windows` on linux so we'd have to make that conditional.

Reply:
I managed to remove the vty-windows dependency. Now everything seems to work with just vty-crossplatform.